src/platform/forms/preSubmitInfo.js

```diff
@@ -12,9 +12,13 @@ export default {
   ),
   field: 'privacyAgreementAccepted',
   label: (
-    <span>
+    <span aria-label="I have read and accept the privacy policy">
       I have read and accept the{' '}
-      <a target="_blank" href="/privacy-policy/">
+      <a
+        aria-label="Privacy policy, will open in new tab"
```

Comment:
@1Copenut  or @noahgelman, is there a downside to using `aria-label` to indicate `will open in new tab`?

Reply:
No, that appears to be a reasonable way to announce links are opening in a new tab. The one downside is that Google Translate may not switch `aria-label` to Spanish. We're moving in a direction where there will be more Spanish content, so knowing these accessible labels are there will be important in any future translations.